src/screens/Tab/DashboardScreen.tsx

```diff
@@ -1,137 +1,263 @@
             </View>
-            <View style={styles.miscContainer}>
-                <Text>Miscellaneous, any ideas what to display here?</Text>
+        </TouchableOpacity>
+    );
+
+    const onCameraPressHandler = async () => {
+        const { status } = await ImagePicker.requestCameraRollPermissionsAsync();
+        status !== 'granted' &&
+            alert('Sorry, we need camera roll permissions to make this work!');
+
+        let result = await ImagePicker.launchImageLibraryAsync({
+            mediaTypes: ImagePicker.MediaTypeOptions.Images,
+            allowsEditing: true,
+            aspect: [4, 3], //android only
+            quality: 1,
+        });
+
+        if (!result.cancelled) {
+            setImage(result.uri); //why is ts angry at me :(
+            //TODO save image to db
+        }
+    };
+
+    const renderRequestCarousel = ({ item }) => (
+        <TitledCard title="Received" style={{ marginHorizontal: 20 }}>
+            <View style={{ height: 200, width: 160 }}>
+                <Text>{item.message}</Text>
+                <Text>
+                    {item.requesterFirstName} {item.requesterLastName}
+                </Text>
             </View>
-        </Layout>
+        </TitledCard>
+    );
+
+    return (
+        <View style={styles.container}>
+            <ScrollView style={{ ...styles.container }} bounces={false}>
+                {/* <Image source={profilePic} style={styles.backgroundImage} /> */}
+                <View style={styles.profileCard}>
+                    <Image
+                        source={image ? { uri: image } : profilePic}
+                        style={styles.profilePictureContainer}
+                    />
+
+                    <TouchableOpacity
+                        style={{ position: 'absolute', left: 130, top: 15 }}
+                        onPress={onCameraPressHandler}
+                    >
+                        <Icon name="camera-outline" {...largeIconProps} />
+                    </TouchableOpacity>
+                    <View
+                        style={{
+                            ...styles.center,
+                            flexDirection: 'column',
+                            height: '100%',
+                            width: '60%',
+                            justifyContent: 'space-around',
+                        }}
+                    >
+                        <Text category="h5" style={styles.profileText}>
+                            Michelle Lin
+                        </Text>
+                        <Text category="h6" style={styles.profileText}>
+                            Philadelphia
+                        </Text>
+                        <Text category="p1" style={styles.profileText}>
+                            Stuff that I do down here
+                        </Text>
+                    </View>
+                </View>
+                <View style={{ ...styles.mainCard, alignItems: 'center' }}>
```

Comment:
So I'm guessing this doesn't work on the react-native `View`?:
```suggestion
                <View className={classes.mainCard}>
```
```js
mainCard: {
  alignItems: 'center',
  // your other styles
},
```

Reply:
I'm making a new style object by reusing styles.mainCard, and tacking on alignItems: 'center'

Its a more shorthand way of doing it, I felt that it wasn't worth making another style object just to add one line, but do you think its better to just make the new object? I can see it being a bit more readable

Reply:
You don't have to make another style object. What I mean is you can just move the `alignItems: 'center'` out of `style` and into your existing `mainCard` lower down in this file: lines [237-244](https://github.com/House-on-a-Rock/tjc-scheduling-app/pull/45/files/cf6c2657208bcf5731b0b3023b4f0035414caaab#diff-15f4359dddf71f58a04dda1650e0f9d3be3a944eac75fb62af6cba9692336491R237-R244)